tests/zephyr/drivers/flash/common/single.overlay

```diff
@@ -0,0 +1,3 @@
+&mx25r64 {
+	mspi-io-mode = "MSPI_IO_MODE_SINGLE";
+};
```

Comment:
Won't it be a problem, that `SDP_MSPI_DQ2` and `SDP_MSPI_DQ3` are defined also for single mode?